Functions/BIDS/xASL_bids_MergeNifti.m

```diff
@@ -659,18 +659,15 @@
             [jsonPathX, jsonNameX] = xASL_fileparts(NiftiPaths{iFileCheck});
             if exist(fullfile(jsonPathX, [jsonNameX '.json']),'file')
                 tmpCheckJSON = spm_jsonread(fullfile(jsonPathX, [jsonNameX '.json']));
-                % Check EchoTimes
-                if isfield(tmpCheckJSON,'SeriesDescription')
-                    isHadamardFME = ~isempty(regexp(char(tmpCheckJSON.SeriesDescription),...
-                                    '(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once')) ... % ASL format
-                                 || ~isempty(regexp(char(tmpCheckJSON.SeriesDescription),...
-                                    '(ss_TE)\d\d(_TI)\d\d\d\d', 'once'));                            % M0 format
-                    if isHadamardFME
-                        if isfield(tmpCheckJSON,'EchoTime')
-                            EchoTimes{iFileCheck,1} = tmpCheckJSON.EchoTime;
-                        end
-                    end
-                end
+                % Check if FME Hadamard
+				isHadamardFME = xASL_imp_CheckIfFME(tmpCheckJSON);
```

Comment:
Should this be an `imp` function (ExploreASL-specific) or a `bids` function (BIDS-specific)?

Reply:
Also, you replaced "check EchoTimes" by "FME Hadamard": that's not generalizable right? What if we have a non-Hadamard non-FME sequence with multiple echotimes?

Reply:
`imp` - it's really needed for general Import.

This is supposed to be non-generalizable. Cause it is dealing with the peculiarities of FME-BBB-ASL. For further multi-TE sequences, we will have to first check if they work right away (as they should) or if they need a special tweak.

Reply:
OK then please call this section (line 649) "FME Hadamard sequence-specific fixes/checks" rather than "If at the end and all went well"

Just for clarity, a section is either 1) general (checking different type of non-standard sequences) 2) Hadamard (checking different Hadamard sequences) or 3) FME-specific

Reply:
Done